client/components/chart/bar.jsx

```diff
@@ -12,7 +12,12 @@ import Gridicon from 'gridicons';
 /**
  * Internal dependencies
  */
-import Tooltip from 'components/tooltip';
+import userModule from 'lib/user';
+
+/**
+ * Module variables
+ */
+const user = userModule();
```

Comment:
We should gradually get rid of the `lib/user` module. In this case, it could be replaced by the Redux value from the `isRtl` selector.